### DISCUSSION TITLE
What is the latest jQuery version is Semantic UI compatible with?

### DISCUSSION THREAD

**Discussion by ronganEnerknol** [fomantic/Fomantic-UI#2852] [General] [answered]
My project was originally installed with **Semantic UI 2.4.2** and **Jquery 1.12.1**. However, after I upgraded my jQuery to **3.6.1**, search and selection dropdowns became un-clickable. No other issues have been found yet. When I downgraded jQuery to **3.1.1,** the search and selection dropdowns worked properly again.

What is the latest version of jQuery is Semantic UI 2.4.2 compatible with?

**Comment by lubber-de** [comment, marked as answer]
We fixed several issues starting with jquery 3.4.0 in FUI, so i guess, 3.3.1 will work with SUI.
Anything else needs to be answered, supported (and possibly fixed) by the original author of SUI 😉 Please use the discussions over at the SUI repo .... or switch to Fomantic 😉
